clients/ethereum/client_test.go

```diff
@@ -220,14 +221,14 @@ func TestClient_GetBatch(t *testing.T) {
 							TokenAddress: token1,
 							Amount:       big.NewInt(20),
 							Depositor:    from1,
-							Recipient:    core.ConvertFromByteSliceToArray(recipient1),
+							Recipient:    data.NewAddressFromBytes(recipient1).AddressSlice(),
```

Comment:
`recipient1.AddressSlice()` and make 
`recipient1 := testsCommon.CreateRandomElrondAddress()`

valid also for recipient 2

Reply:
done